pyquil/api/_qpu.py

```diff
@@ -72,6 +72,9 @@ def __init__(self, endpoint: str, user: str = "pyquil-user") -> None:
         self.user = user
         self._last_results: Dict[str, np.ndarray] = {}
 
+    def get_version_info(self) -> dict:
```

Comment:
What does it mean for a QPU to have a version. I presume this actually will get the version of the classical host server. Can you document this?